config/install/image.style.media_library_item.yml

```diff
@@ -0,0 +1,6 @@
+langcode: en
+status: true
+dependencies: {  }
+name: media_library_item
+label: 'Media library item'
+effects: {  }
```

Comment:
It might interfere with the masonry layout. Just review it once without the thumbnail effect. If you still feel the need, I'll do it.